tests/test_test_utils.py

```diff
@@ -0,0 +1,143 @@
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from dwave.plugins.torch.nn import store_config
+from tests import utils
+
+
+class TestUtils(unittest.TestCase):
```

Comment:
Not sure I like the `test_test_utils.py` name. Maybe better to call this something different. Perhaps "helper_functions"? Not sure. Or simply call the file `test_utilities.py` and have it contain both the utility function _and_ it's tests. Or rename the other `test_utils.py` to `test_torch_utils.py`.